TodoApp.ConsoleApp/tasks.xml

```diff
@@ -12,4 +12,10 @@
     <dateCreated>29 04 2019</dateCreated>
     <deadline>01 05 2019</deadline>
   </task>
+  <task id="1bd895a6-a633-45ff-9419-625774b9498d">
```

Comment:
Add `tasks,xml` to `.gitignore`.
We don't need to track changes in this file. We care only for the code and the project. The data itself is generated from the usage of the app.